setup.cfg

```
@@ -40,3 +40,7 @@ exclude_lines =
 	raise NotImplementedError
 
 	if __name__ == .__main__.:
+
+[i18n]
+project = kolibri
+locale_data_folder = kolibri/locale
```

could do something along the lines of...


```suggestion
[kolibri:i18n]
project = kolibri
locale_data_folder = kolibri/locale
```

where eventually we'd also have e.g.

```ini
[kolibri:i18n]
project = studio
locale_data_folder = contentcuration/locale
```

---

Hrm, not a big fan of the `:`, but it seems like Python ini parsers that allow subsections (ConfigObj) do it in a completely different way to the ini file parser for nodejs that I'm using, so this might be simplest.